.github/workflows/cancel.yml

```diff
@@ -0,0 +1,29 @@
+name: Auto-Cancel
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+    types:
+      - opened
+      - closed
+      - reopened
+      - synchronize
+
+jobs:
+  cancel:
+    name: 'Cancel Previous Runs'
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      # See https://github.com/marketplace/actions/cancel-workflow-action#advanced
+      - uses: styfle/cancel-workflow-action@0.6.0
+        env:
+          # See https://api.github.com/repos/google/site-kit-wp/actions/workflows for reference.
+          WID_SB: 1979909 # storybook.yml
+          WID_VRT: 2751946 # visual-regression.yml
+          WID_ZIPS: 1979910 # zips.yml
```

Comment:
According to the docs, this also supports workflow names - wouldn't this be more intuitive to use?

Reply:
Looking at it again, that would be okay too. I originally read the docs for that to mean you could use the actual workflow `name` which I wouldn't want but it's the file name which is much more stable.

I guess it depends on how GH treats the workflow ID in relation to the file name. I would expect the ID to remain unchanged even if the workflow changed completely.

After taking a closer look at this, I found the [API documentation](https://developer.github.com/v3/actions/workflow-runs/#list-workflow-runs) for listing workflow runs which is what this uses (via octokit). 

> List all workflow runs for a workflow. You can also replace `:workflow_id` with `:workflow_file_name`. For example, you could use `main.yml`. You can use parameters to narrow the list of results. For more information about using parameters, see Parameters.

So that means we wouldn't need to use the full `.github/workflows/zips.yml` path which is what I was thinking it would require.

File names it is then? 😄

Reply:
Updated to use filenames instead. It's much cleaner and doesn't require referencing something external to update. Also, the ability to use a filename instead is a feature provided by the underlying GitHub API ([example](https://api.github.com/repos/google/site-kit-wp/actions/workflows/storybook.yml/runs)) rather than the extension so this is much better I think 👍